aws/sdk/examples/transcribestreaming/src/main.rs

```diff
@@ -4,27 +4,72 @@
  */
 
 use async_stream::stream;
-
 use aws_config::meta::region::RegionProviderChain;
 use aws_sdk_transcribestreaming::model::{
     AudioEvent, AudioStream, LanguageCode, MediaEncoding, TranscriptResultStream,
 };
-use aws_sdk_transcribestreaming::{Blob, Client, Error, Region};
+use aws_sdk_transcribestreaming::{Blob, Client, Error, Region, PKG_VERSION};
 use bytes::BufMut;
 use std::time::Duration;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+struct Opt {
+    /// The AWS Region.
+    #[structopt(short, long)]
+    region: Option<String>,
+
+    /// The name of the audio file.
+    #[structopt(short, long)]
+    audio_file: String,
+
+    /// Whether to display additional information.
+    #[structopt(short, long)]
+    verbose: bool,
+}
 
 const CHUNK_SIZE: usize = 8192;
 
+/// Transcribes an audio file to text.
+/// # Arguments
+///
+/// * `-a AUDIO_FILE` - The name of the audio file.
+///   It must be a WAV file.
```

Comment:
Probably good to call out that this is a limitation of the example and not of Transcribe.

Reply:
Good point. I'll add a note that mentions the supported formats.